src/Compilers/CSharp/Test/Symbol/Symbols/SymbolExtensionTests.cs

```diff
@@ -84,6 +87,198 @@ class C { }
+        instance void .ctor () cil managed 
+    {
+        // Method begins at RVA 0x2057
+        // Code size 7 (0x7)
+        .maxstack 8
+
+        IL_0000: ldarg.0
+        IL_0001: call instance void System.Object::.ctor()
+        IL_0006: ret
+    } // end of method C1`1::.ctor
+
+} // end of class C1`1
+
+.class public auto ansi beforefieldinit C2`1<T>
+    extends System.Object
+{
+    // Methods
+    .method public hidebysig specialname rtspecialname 
+        instance void .ctor () cil managed 
+    {
+        // Method begins at RVA 0x2057
+        // Code size 7 (0x7)
+        .maxstack 8
+
+        IL_0000: ldarg.0
+        IL_0001: call instance void System.Object::.ctor()
+        IL_0006: ret
+    } // end of method C2`1::.ctor
+
+} // end of class C2`1
+
+.class public auto ansi beforefieldinit C3`1<T>
+    extends class C1`1<int32 modopt(class C2`1<!T>)>
+{
+    // Methods
+    .method public hidebysig specialname rtspecialname 
+        instance void .ctor () cil managed 
+    {
+        // Method begins at RVA 0x205f
+        // Code size 7 (0x7)
+        .maxstack 8
+
+        IL_0000: ldarg.0
+        IL_0001: call instance void class C1`1<int32>::.ctor()
+        IL_0006: ret
+    } // end of method C3`1::.ctor
+
+} // end of class C3`1
+";
+
+            var source = @"
+class Test
+{
+    static void Main()
+    {
+        M<int>();
+    }
+
+    static void M<G>()
+    {
+        System.Func<string> x = C3<G>.Method;
+        System.Console.WriteLine(x());
+    }
+}
+";
+            var compilation = CreateCompilationWithIL(source, ilSource, options: TestOptions.ReleaseExe);
+
+            var tree = compilation.SyntaxTrees.Single();
+            var model = compilation.GetSemanticModel(tree);
+            var method = model.GetSymbolInfo(tree.GetRoot().DescendantNodes().OfType<IdentifierNameSyntax>().Where(id => id.Identifier.ValueText == "Method").Single()).Symbol.GetSymbol<MethodSymbol>();
+
+            AssertEx.Equal("System.String C1<System.Int32 modopt(C2<G>)>.Method()", method.ToTestDisplayString());
+
+            var typeParameters = PooledHashSet<TypeParameterSymbol>.GetInstance();
+            try
+            {
+                method.ContainingType.VisitType(static (typeSymbol, typeParameters, _) =>
+                {
+                    if (typeSymbol is TypeParameterSymbol typeParameter)
+                    {
+                        typeParameters.Add(typeParameter);
+                    }
+
+                    return false;
+                },
+                typeParameters, visitCustomModifiers: true);
+
+                var typeParameter = typeParameters.Single();
+                Assert.Equal("G", typeParameter.Name);
+                Assert.Equal("M", typeParameter.ContainingSymbol.Name);
+            }
+            finally
+            {
+                typeParameters.Free();
+            }
+
+            //CompileAndVerify(compilation, expectedOutput: @"Method");
```

Comment:
>//CompileAndVerify(compilation, expectedOutput: @"Method"); [](http://example.com/codeflow?start=12&length=59)

Consider removing this commented out line.

Reply:
Sure.